.travis.yml

```diff
@@ -18,17 +18,22 @@ matrix:
       addons:
         apt:
           packages:
-            - python3-dev
-      env: PYTHONVERSION=python3
+            - python3.4-dev
+      env: PYTHONVERSION=python3.4
     - compiler: gcc
-      python: "3.5"
+      python: "3.6.1"
       dist: trusty
       addons:
         apt:
           packages:
-            - python3-dev
+            - python3
       env: PYTHONVERSION=python3
 
+before_install:
+  - sudo add-apt-repository ppa:jonathonf/python-3.6 -y
```

Comment:
ubuntu trusty 14.04 doesn't have an official ppa for python3.6 so i needed to do this to get the dev artifacts